assets/airports/airportLoadList.json

```diff
@@ -172,6 +172,12 @@
         "name": "Nashville International Airport",
         "premium": false
     },
+    {
+        "icao": "kchs",
+        "level": "easy",
```

Comment:
With the current traffic levels, this qualifies as a "hard" airport. I would recommend you decrease the traffic to below 40acph to make it "easy".